README.md

```diff
@@ -4,110 +4,64 @@
   # Emu-MPS
 </div>
 
-**EMU-MPS** is a Pulser backend, designed to **EMU**late the dynamics of programmable arrays of neutral atoms, with matrix product states (**MPS**). It allows users to increase the number of qubits and reduce computation time.
+**EMU-MPS** is a backend for the [Pulser low-level Quantum Programming toolkit](https://pulser.readthedocs.io). EMU-MPS lets you transparently run Quantum Algorithms on a simulated device, using GPU acceleration if available. More in depth, EMU-MPS is designed to **EMU**late the dynamics of programmable arrays of neutral atoms, with matrix product states (**MPS**). This design makes it faster and more memory-efficient than previous generations of quantum emulators, which means that you can emulate systems with larger number of qubits, faster.
```

Comment:
It will be faster above a certain threshold due to exponential scaling of state vector emulators. The above implies that it's faster always, which is definitely not true. Perhaps we should also hedge a bit and compare with state vector emulators specifically.

Reply:
Reformulated, what do you think?